docs/topics/installing-vs-code-extension.adoc

```diff
@@ -10,7 +10,12 @@ You can install the {ProductShortName} extension for Visual Studio Code (VS Code
 .Prerequisites
 
 * 8 GB RAM
-* OpenJDK 1.8, OpenJDK 11, Oracle JDK 1.8, _or_ Oracle JDK 11
+* One of the following Java Development Kits (JDKs):
```

Comment:
The prerequisites would be a perfect candidate for a separate module! It does not have to part of this PR, however.